indico/modules/events/contributions/forms.py

```diff
@@ -46,7 +47,8 @@ def __init__(self, *args, **kwargs):
 
 class ContributionProtectionForm(IndicoForm):
     protection_mode = IndicoEnumRadioField(_('Protection mode'), enum=ProtectionMode)
-    acl = PrincipalListField(_('Access control list'), [UsedIf(lambda form, field: form.contrib.is_protected)],
+    protection_message = IndicoProtectionMessage(_('Protection message'))
```

Comment:
I don't think this is a good idea. It's not really a field. Why not create a widget that contains the radios for the protection mode and the message below it (in the same widget)?